app/src/main/java/com/eveningoutpost/dexdrip/LibreAlarmReceiver.java

```diff
@@ -210,9 +210,15 @@ public static void CalculateFromDataTransferObject(ReadingData.TransferObject ob
                 Pref.setBoolean("nfc_age_problem", false);
                 Log.d(TAG, "Sensor age advanced to: " + thisSensorAge);
             } else if (thisSensorAge == sensorAge) {
-                Log.wtf(TAG, "Sensor age has not advanced: " + sensorAge);
-                JoH.static_toast_long(gs(R.string.sensor_clock_has_not_advanced));
-                Pref.setBoolean("nfc_age_problem", true);
+                // This is only a problem if we don't have a recent reading. It could happen that we have a recent
+                // reading, and then BT disconnects and connects after a few seconds, and we have a new reading (where 
+                // sensor did not advance). This does not mean that a sensor is not advancing. Only if this is happening
+                // for a few minutes, this is a problem.
+                if(BgReading.getTimeSinceLastReading() > 11 * 60 * 1000) {
```

Comment:
This looks like it should work, by the way you can use the `msSince()` helper and also `Constants.MINUTE_IN_MS` to improve readability.